notebooks/official/pytorch_image_classification_distributed_data_parallel_training_with_vertex_sdk/multi_node_ddp_gloo_vertex_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<p>It is a large database of handwritten digits that is commonly used for training various image processing systems.  -&gt; replace this with official description.</p><p> </p><p>http://yann.lecun.com/exdb/mnist/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>